src/main/archetype/ui.content/src/main/content/jcr_root/conf/__confFolderName__/settings/wcm/templates/experience-fragment-page/initial/.content.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        cq:lastModified="{Date}2020-01-10T13:53:38.591+02:00"
+        cq:lastModifiedBy="admin"
+        cq:template="/conf/${confFolderName}/settings/wcm/templates/experience-fragment-venia-demo-store"
```

Comment:
there is still "venia" hardcoded in the demo-store name. Isn't the name wrong here?... I cannot find this anywhere in your commit, so where is it coming from?

Reply:
It's the name of the template. I'll update the name.